File: MultiplesOfANumber/multiples.rb
# multiples of a number solution in ruby for code eval by steven a dunn

File.open(ARGV[0]).each_line do |line|
  x, n = line.split(',').map {|x| x = x.to_i }
  
  multiplier = 2
  while true
    multiple = n * multiplier
    if multiple >= x
      puts multiple
      break
    end
    multiplier += 1
  end
end

File: MultiplesOfANumber/py2/multiples.py
# Multiples of a Number solution in Python 2 for CodeEval.com by Steven A. Dunn

import sys

f = open(sys.argv[1], 'r')

for line in f:
  x, n = line.split(',')
  x = int(x)
  n = int(n)

  multiple = 2
  multipleN = n
  while x >= multipleN:
    multipleN = n * multiple
    multiple += 1
  print multipleN
